src/steps/account/account-field-equals.ts

```diff
@@ -31,7 +31,7 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
     optionality: FieldDefinition.Optionality.OPTIONAL,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
+    description: 'Check Logic (set to, not set to, containing, not containing, greater than, less than, be set, or not be set)',
   }, {
     field: 'expectedValue',
     type: FieldDefinition.Type.ANYSCALAR,
```

Comment:
- [ ] Since `be set` and `not be set` have no expected values, this will need to be marked as optional.